domain-check-2.sh

```diff
@@ -414,7 +414,7 @@ check_domain_status()
     fi
     if [ "${TLDTYPE}" == "ca" ]; # added by @hawkeye116477 20190511
```

Comment:
Remove completely 415-418 lines.
Without them, I have the .ca zone defined correctly.
If you are wrong, show the output:
```
sh domain-check-2.sh -v -d google.ca
```
or
```
bash domain-check-2.sh -v -d google.ca
````

Reply:
Yeah, you are right.